### ISSUE TITLE
notifier version is never processed in grunt template

### ISSUE THREAD

**Issue by clayzermk1** [airbrake/airbrake-js#179]
Airbrake notifications have a context.notifier like the following:
```javascript
{
    "name": "airbrake-js-v8-short",
    "version": "<%= pkg.version %>",
    "url": "https://github.com/airbrake/airbrake-js"
}
```

The source has what appears to be a grunt template that is never being processed.
https://github.com/airbrake/airbrake-js/blob/9fa33713ad9d5204e29b287cf56417a4e3e85467/src/client.coffee#L132
https://github.com/airbrake/airbrake-js/blob/39534e41def9f3bd727b1ec54fcebf87645e7877/Gruntfile.coffee#L9

**Comment by vmihailenco** [comment]
We don't distribute coffee files, but we distribute JS files which have that var properly processed https://github.com/airbrake/airbrake-js/blob/master/dist/client.js#L177

**Comment by clayzermk1** [comment]
Huh, sure enough. I think the "problem" here is that I'm using npm to get airbrake-js which has the [`main` set to `lib/client.js`](https://github.com/airbrake/airbrake-js/blob/master/package.json#L48). I'll have to tweak my includes a bit to work around that. Is there a reason this points to the lib and not the dist?

**Comment by vmihailenco** [comment]
AFAIK the theory behind this is following:
- lib contains source that can be included with RequireJS
- dist contains code compiled with Browserify (can be included directly using `<script>`)

So I think we need to fix lib/client.js to contain proper version.

**Comment by vmihailenco** [comment]
Should be fixed in v0.5.7. Thanks for digging this for us!

**Comment by clayzermk1** [comment]
Awesome! I'll give it a go. Thank you :smile:
